deploy/terraform-datacommons-website/modules/bt_cache/variables.tf

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+variable "resource_suffix" {
+  type        = string
+  description = "Optional suffix to add to GCP resources for uniqueness."
+  default     = null
+}
+
+variable "project_id" {
+  type        = string
+  description = "This is the same GCP project id from the setup step."
+}
+
+variable "region" {
```

Comment:
Rather than defining the default at the module level, the default will be defined in the instance (examples folder) level and passed down. Same for other variables that are shared.